### PR TITLE
Add emphasis to asking price and datetime labels in GUI

### PR DESCRIPTION
**Pull request by w2vgd**
Open to suggestion for color palette 😅 

Current example:
image

### CODE REVIEW

**Comment by dvdweien** [comment]
Would bold or underline for the date time label work better? Personally I find that red is too eye-catching for me

File: src/main/resources/view/LightTheme.css [hunk 1]
```diff
@@ -367,3 +367,27 @@
     -fx-font-size: 13px;
     -fx-text-fill: black;
 }
+
+#deadline {
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-font-size: 14px;
+    -fx-text-fill: tomato;
```
Comment (dvdweien) [inline src/main/resources/view/LightTheme.css:374]: maybe only set the text to red when the time has passed like you have already done in appointment and property card?

Comment (dvdweien) [inline src/main/resources/view/LightTheme.css]: maybe only set the text to red when the time has passed like you have already done in appointment and property card?

**Review by dvdweien — COMMENTED** [review]
looks good overall, just a suggestion, maybe only set the text to red when the time has passed without making the default colour red as well?

**Comment by w2vgd** [comment]
Do you mean like this? Is this better?

image

**Comment by dvdweien** [comment]
i think can lose the underline, maybe the bold isn't needed oso. I was just confused why every date was red

**Comment by w2vgd** [comment]
> i think can lose the underline, maybe the bold isn't needed oso. I was just confused why every date was red

Then wont it be the same as the original alr? The main idea was to place some emphasis on the dates so that the user can see his upcoming deadlines/appointments more clearly

**Comment by dvdweien** [comment]
in that case, i think just bold is enough underline looks a little ugly to me

**Comment by w2vgd** [comment]
Updated image:
image

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**